drivers/volume/kdmp/kdmp.go

```diff
@@ -180,6 +188,94 @@ func isCSISnapshotClassRequired(pvc *v1.PersistentVolumeClaim) bool {
 	return !storkvolume.IsCSIDriverWithoutSnapshotSupport(pv)
 }
 
+func getRegionalZones(zones string) []string {
+	return strings.Split(zones, zoneSeperator)
+}
+
+func isRegional(zone string) bool {
+	return strings.Contains(zone, zoneSeperator)
+}
+
+func getZones(pv *v1.PersistentVolume) ([]string, error) {
```

Comment:
Suggestion: Will it be good, if we move this getZones to be a common code between the actual cloud driver and kdmp driver. Some thing like moving it in to volume.go. That if any fixes needed can be done in one place.